Cleanup files: Remove trailing whitespace, Ensure all text files end with \n

ypid-geberit opened the pull request:
<!--
Thank you for your interest in and contributing to ECS! There are a
few simple things to check before submitting your pull request that
can help with the review process. You should delete these items from
our submission, but they are here to help bring them to your attention.
-->

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/ecs/blob/master/CONTRIBUTING.md)?
- For proposing substantial changes or additions to the schema, have you reviewed the [RFC process](https://github.com/elastic/ecs/blob/master/rfcs/README.md)?
- If submitting code/script changes, have you verified all tests pass locally using `make test`?
- If submitting schema/fields updates, have you generated new artifacts by running `make` and committed those changes?
- Is your pull request against master? Unless there is a good reason otherwise, we prefer pull requests against master and will backport as needed.
- Have you added an entry to the [CHANGELOG.next.md](https://github.com/elastic/ecs/blob/master/CHANGELOG.next.md)?

I am not doing this out of boredom, it is rather that editors might clean whitespace on save which creates commits that are not atomic anymore, unless you use `git add -i` or something which causes extra work.

I came across this while I was working on my org specific patchset I maintain ontop of the last ECS release, ref: https://github.com/elastic/ecs/issues/443#issuecomment-818633561

ebeahan commented:
@elasticmachine run elasticsearch-ci/docs

ebeahan reviewed (commented):
Thanks for the cleanup, @ypid-geberit. The motivations for cleaning these files make sense.

Can you restore the .PNG file modified [here](https://github.com/elastic/ecs/pull/1564/files#diff-e0ea776405f08380b9e241c5d5e7c6fe6517f40094fccd08f24d469a23fbbae1)? Whatever was change is causing the image not to render in the docs preview for this PR.

Instead of fixing the whitespace after the fact with `.strip()`, another possibility is fixing the Jinja2 templates to render without the excess whitespace from the start.

I made some adjustments in a separate branch as examples. Take a look at these commitments and let me know what you think:

* https://github.com/ebeahan/ecs/commit/4cc2008ddced441bac86f13dc2372e254bb54204
* https://github.com/ebeahan/ecs/commit/d2bd328c5854787c7866e53bb91ef725e40be001
* https://github.com/ebeahan/ecs/commit/250f81032f402034e1aeab17b5d840626392496f
* https://github.com/ebeahan/ecs/commit/c09cb133bd27cb96f40b2ee72649bf4aa03884e8

ypid-geberit commented:
You are welcome.

I reverted the PNG. I should have excluded png files in that find command also as I did for the other.

Makes sense to avoid producing white space at the root instead of fixing it afterwards. I cherry picked your commits, rebased and finished it. Can you have a second look?

ypid-geberit commented:
I resolved the merge conflicts. Can you reconsider this PR? Leaving it unmerged creates merge conflicts.
